src/structures/Guild.js

```diff
@@ -504,6 +504,16 @@ class Guild {
   }
 
   /**
+   * Adds a bot user to the Guild, this will not work for bot accounts.
+   * @param {string} clientID The Client ID of the bot, this isn't always the bot's User ID
+   * @param {number} [permissions=0] An optional permissions number to give the bot upon authorization
+   * @returns {Promise<Object>}
+   */
+  addBot(clientID, permissions = 0) {
```

Comment:
Please allow permissions to also take an array of [PermissionResolvables](https://discord.js.org/#!/docs/tag/master/typedef/PermissionResolvable), it makes it easier for the end user.